src/sentry/migrations/0532_add_notification_uuid_to_rule_fire_history.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.20 on 2023-08-16 17:55
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
```

Comment:
@jangjodi shouldn't we merge this first separately?

Reply:
It can be safer, if we want to split up the migration and the rest of this pr that could make sense

Reply:
Thank you for the suggestion - I've opened a PR just for the migration here https://github.com/getsentry/sentry/pull/54888